package-lock.json

```diff
@@ -3328,9 +3328,9 @@
       "dev": true
     },
     "chromedriver": {
-      "version": "76.0.1",
-      "resolved": "https://registry.npmjs.org/chromedriver/-/chromedriver-76.0.1.tgz",
-      "integrity": "sha512-+8BCemJLKPF2w/UpzA1uNgLWQrg1IgIO4ZYcsAjYYgqD8zUcvQ+RfwA/0TR1Zwv9Mkd8fdzTe21eZ2FyZ83DAg==",
+      "version": "77.0.0",
```

Comment:
Revert file?

Reply:
It's part of a different fix, just needed to check that it'd work in CI. Will rebase it out